src/ui/EditorNS/editor.cpp

```diff
@@ -678,52 +778,16 @@ namespace EditorNS
         sendMessage("C_CMD_SET_TABS_VISIBLE", visible);
     }
 
-    Editor::IndentationMode Editor::detectDocumentIndentation(bool *found)
-    {
```

Comment:
This was literally only used for banner displays, which now operates on a signal rather than a function call.  We can always re-implement it later if the need arises (over-engineering is the devil).